app/src/main/kotlin/com/strv/archdemo/ui/counter/CounterScreen.kt

```diff
@@ -0,0 +1,248 @@
+
+            Counter(
+                state = state,
+                onDecreaseCounter = onDecreaseCounter,
+                onIncreaseCounter = onIncreaseCounter
+            )
+
+            Network(
+                state = state,
+                onBlockingCall = onBlockingCall,
+                onAsyncCall = onAsyncCall,
+            )
+        }
+    }
+}
+
+@Composable
+private fun CounterTopBar(onBack: () -> Unit) {
+    TopAppBar(
+        title = {
+            Text(text = "CounterScreeen")
+        },
+        navigationIcon = {
+            IconButton(
+                onClick = onBack,
+                modifier = Modifier.clip(CircleShape)
+            ) {
+                Icon(imageVector = Icons.Default.ArrowBack, contentDescription = "Back")
+            }
+        }
+    )
+}
+
+@Composable
+private fun Header(state: CounterViewState) {
+    Text(
+        text = "CounterScreen",
+        style = MaterialTheme.typography.h3,
+        modifier = Modifier.padding(top = 24.dp)
+    )
+    Text(
+        text = state.title,
+        style = MaterialTheme.typography.h5,
+        modifier = Modifier.padding(top = 16.dp)
+    )
+}
+
+@Composable
+private fun Counter(
+    state: CounterViewState,
+    onDecreaseCounter: () -> Unit,
+    onIncreaseCounter: () -> Unit
+) {
+    Row {
+        Text(
+            text = "Click count: ",
+            style = MaterialTheme.typography.body1,
+            modifier = Modifier.padding(top = 48.dp)
+        )
+
+        Text(
+            text = "${state.clickCount}",
+            style = MaterialTheme.typography.body1.copy(fontWeight = FontWeight.Bold),
+            modifier = Modifier.padding(top = 48.dp)
+        )
+    }
+    Row(
+        horizontalArrangement = Arrangement.SpaceEvenly,
+        modifier = Modifier.fillMaxWidth()
+    ) {
+
+        IconButton(
+            onClick = onDecreaseCounter,
+            modifier = Modifier
+                .padding(top = 48.dp)
+                .clip(MaterialTheme.shapes.small)
+                .background(MaterialTheme.colors.primary)
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_minus_1),
+                contentDescription = "Decrease counter",
+                tint = contentColorFor(backgroundColor = MaterialTheme.colors.primary),
+                modifier = Modifier.padding(20.dp)
+            )
+        }
+
+        IconButton(
+            onClick = onIncreaseCounter,
+            modifier = Modifier
+                .padding(top = 48.dp)
+                .clip(MaterialTheme.shapes.small)
+                .background(MaterialTheme.colors.primary)
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_plus_1),
+                contentDescription = "Increase counter",
+                tint = contentColorFor(backgroundColor = MaterialTheme.colors.primary),
+                modifier = Modifier.padding(20.dp)
+            )
+        }
+    }
+}
```

Comment:
Just move paddings to parent Rows. I want to add there content animation and then row is not wrapping count number, it's messing the animation up 
```suggestion
@ExperimentalAnimationApi
@Composable
private fun Counter(
    state: CounterViewState,
    onDecreaseCounter: () -> Unit,
    onIncreaseCounter: () -> Unit
) {
    Row(modifier = Modifier.padding(top = 48.dp)) {
        Text(
            text = "Click count: ",
            style = MaterialTheme.typography.body1
        )
        Text(
            text = "${state.clickCount}",
            style = MaterialTheme.typography.body1.copy(fontWeight = FontWeight.Bold),
        )
    }
    Row(
        horizontalArrangement = Arrangement.SpaceEvenly,
        modifier = Modifier
            .padding(top = 48.dp)
            .fillMaxWidth()
    ) {
        IconButton(
            onClick = onDecreaseCounter,
            modifier = Modifier
                .clip(MaterialTheme.shapes.small)
                .background(MaterialTheme.colors.primary)
        ) {
            Icon(
                painter = painterResource(id = R.drawable.ic_minus_1),
                contentDescription = "Decrease counter",
                tint = contentColorFor(backgroundColor = MaterialTheme.colors.primary),
                modifier = Modifier.padding(20.dp)
            )
        }

        IconButton(
            onClick = onIncreaseCounter,
            modifier = Modifier
                .clip(MaterialTheme.shapes.small)
                .background(MaterialTheme.colors.primary)
        ) {
            Icon(
                painter = painterResource(id = R.drawable.ic_plus_1),
                contentDescription = "Increase counter",
                tint = contentColorFor(backgroundColor = MaterialTheme.colors.primary),
                modifier = Modifier.padding(20.dp)
            )
        }
    }
}
```